fe/src/main/java/org/apache/doris/analysis/DateLiteral.java

```diff
@@ -150,17 +178,28 @@ public String toSqlImpl() {
 
     @Override
     public String getStringValue() {
-        return TimeUtils.format(date, type);
+        if (type == Type.DATE) {
```

Comment:
I think we use one format is enough?
String.format("%04d-%02d-%02d", year, month, day)?